apps/server/internal/httpapi/question_get_test.go

```
@@ -0,0 +1,59 @@
+		Kind:      "TEXT",
+		Blocking:  true,
+		Status:    "OPEN",
+	}}}
+	questionService, err := app.NewQuestionService(questionStore)
+	if err != nil {
+		t.Fatal(err)
+	}
+	router := newRouter(app.New(&fakeControlPlaneStore{}), nil, nil, nil, questionService)
+
+	get := httptest.NewRecorder()
+	router.ServeHTTP(get, httptest.NewRequest(http.MethodGet, "/api/projects/"+projectID+"/questions/"+otherID, nil))
```

golangci-lint (2.13.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)